src/coprocessor/codec/table.rs

```diff
@@ -313,29 +320,35 @@ impl RowColsDict {
         RowColsDict { value, cols }
     }
 
+    /// Returns the total count of the columns.
+    #[inline]
     pub fn len(&self) -> usize {
         self.cols.len()
     }
 
+    /// Returns whether has columns or not.
```

Comment:
-> Returns whether columns exist or not.
or
-> Returns whether it has columns or not.